### PR TITLE
Fixing compilation error in saxpy.cpp

### PR DESCRIPTION
**Pull request by kjayapra-amd**
Creating Pull Request to fix missing vector header file in saxpy.cpp.

### CODE REVIEW

**Comment by gargrahul** [comment]
@kjayapra-amd Why was this issue not seen with HIP/HCC testing ?

**Comment by emankov** [comment]
Please, provide your command line and error log.

File: tests/src/hiprtc/saxpy.cpp [hunk 1]
```diff
@@ -83,7 +84,7 @@ int main()
     size_t codeSize;
     hiprtcGetCodeSize(prog, &codeSize);
 
-    vector<char> code(codeSize);
+    std::vector<char> code(codeSize);
```
Comment (mangupta) [inline tests/src/hiprtc/saxpy.cpp]: This should be unnecessary since we already have a using namespace std on line 60.

Comment (kjayapra-amd) [reply tests/src/hiprtc/saxpy.cpp]: Removed the std:;, since whole namespace has been imported.

**Comment by kjayapra-amd** [comment]
@emankov, Pasting the error below.

hipcc-cmd: ../../../../../dist/linux/debug/bin/clang  -std=c++11 -isystem ../../../../../dist/linux/debug/include/clang -fhip-new-launch-api -isystem ../../../../../dist/linux/debug/include -DHIP_VERSION_MAJOR=1 -DHIP_VERSION_MINOR=5 -DHIP_VERSION_PATCH=19245 --cuda-gpu-arch=gfx803 -D__HIP_ARCH_GFX803__=1  --cuda-gpu-arch=gfx900 -D__HIP_ARCH_GFX900__=1  --cuda-gpu-arch=gfx906 -D__HIP_ARCH_GFX906__=1  --cuda-gpu-arch=gfx908 -D__HIP_ARCH_GFX908__=1  --cuda-gpu-arch=gfx1010 -D__HIP_ARCH_GFX1010__=1  --cuda-gpu-arch=gfx1012 -D__HIP_ARCH_GFX1012__=1  -O3  --hip-device-lib-path=../../../../../dist/linux/debug/lib/bitcode  -g -I ../../../src -c -x hip ../../../src/hiprtc/saxpy.cpp -o saxpy.o
../../../src/hiprtc/saxpy.cpp:95:17: error: expected ';' after expression
    vector<char> code(codeSize);
                ^
                ;
../../../src/hiprtc/saxpy.cpp:95:5: error: use of undeclared identifier 'vector'
    vector<char> code(codeSize);
    ^
../../../src/hiprtc/saxpy.cpp:95:18: error: use of undeclared identifier 'code'
    vector<char> code(codeSize);
                 ^
../../../src/hiprtc/saxpy.cpp:96:25: error: use of undeclared identifier 'code'
    hiprtcGetCode(prog, code.data());
                        ^
../../../src/hiprtc/saxpy.cpp:103:32: error: use of undeclared identifier 'code'
    hipModuleLoadData(&module, code.data());
                               ^

**Pull request merged**